### ISSUE TITLE
The feedback is empty if the student shuts down the process with os._exit(0)

### ISSUE THREAD

**Issue by Mankro** [apluslms/python-grader-utils#21]
If the student's submission calls `os._exit(0)`, the final feedback in A+ is completely empty. I assume that the python-grader-utils process shuts down and is not able to produce any output in the end. 

Is the remote grader prototype able to fix this problem? If the grader main process and the submission process are separated, then the student can not shut down the main process.
https://github.com/apluslms/python-grader-utils/tree/remotegrader

**Comment by ihalaij1** [comment]
I got the remotegrader working with our recent updates to graderutils.
When using the remotegrader, `os._exit(0)` results in an `EOFError` when the rpyc connection is lost, but it does not shut down the main process.

`os._exit()` could probably be overridden in the student process so that it raises an exception stating that the command should not be used. Is this too hacky?

**Comment by Mankro** [comment]
I don't think this is urgent. We can now leave this waiting for the remote grader. We will then get a better overview of this whole thing. 

If you are already finished with the remote grader, then I can take a look at that in May.
